Formula/mercurial.rb

```diff
@@ -13,6 +14,14 @@ class Mercurial < Formula
     sha256 "8934a8fc26c36e56f017d99ac43a0ebe9d6922f8276c1dfccd9ccb980ae05e45" => :mavericks
   end
 
+  option "with-custom-python", "Install against the python in PATH instead of "\
+                               "Homebrew's python"
+  if build.with? "custom-python"
```

Comment:
If we can figure out a way to avoid needing the `build.with?` here that'd be good as we all know it causes breakage.